Sources/X/Capability/Types/Genesis.pb.swift

```diff
@@ -0,0 +1,23 @@
+import CosmosProto
+
+// GenesisState defines the capability module's genesis state.
+struct CapabilityGenesisState: Codable {
+    // index is the capability global index.
+    let index: UInt64
+    // owners represents a map from index to owners of the capability index
+    // index key is string to allow amino marshalling.
+    let owners: [GenesisOwners]
+}
+
+// GenesisOwners defines the capability owners with their corresponding index.
+struct GenesisOwners: Codable {
```

Comment:
I think this guy may need to have the @StringBackedInt property wrapper if it is ever used as json.